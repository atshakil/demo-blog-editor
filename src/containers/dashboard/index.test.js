import React from 'react';
import ReactDOM from 'react-dom';
import { VisibleDashboard } from '.';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<VisibleDashboard />, div);
  ReactDOM.unmountComponentAtNode(div);
});
