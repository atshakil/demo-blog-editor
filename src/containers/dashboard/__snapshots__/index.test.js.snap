// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="Dashboard container"
>
  <div
    className="section-a row"
  >
    <div
      className="col"
    >
      <div
        className="photo-uploader row"
      />
      <div
        className="name-editor row"
      />
    </div>
  </div>
  <div
    className="section-b row"
  >
    <div
      className="content-editor col"
    />
  </div>
  <div
    className="section-c row"
  >
    <div
      className="submit-button col"
    />
  </div>
</div>
`;
